src/style.css

```
@@ -0,0 +1,231 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #060b35;
+}
+
+.title {
+  display: none;
+}
+
+.card-container {
+  /* padding-top: 51px; */
+  /* gap: 50px; */
+  /* margin: 0 auto; */
+  max-width: 1200px;
+  display: flex;
+  flex-wrap: wrap;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+}
+
+/* #region menu */
+.home-image-link {
+  height: 40px;
+  width: 40px;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 1200px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.nav-item {
+  margin-right: 20px;
+  list-style: none;
+  position: relative;
+  text-transform: uppercase;
+  text-decoration: none;
+  align-items: flex-end;
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
+
+.nav_tag {
+  display: inline-block;
+  margin: 0;
+  text-decoration: none;
+  font-size: 12px;
+  padding: 22.5px 0;
```

do not use such values, you can round them
```suggestion
  padding: 23px 0;
```